Artem Sushko/Lesson5/Lesson5.Homework/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿//The greatest common divisor
+
+while (true)
+{
+    Console.Clear();
+    Console.Write("Enter first value: ");
+    var x = uint.Parse(Console.ReadLine());
+    Console.Write("Enter second value: ");
+    var y = uint.Parse(Console.ReadLine());
+
+    var z = 0;
+    for (int i = 1; i <= Math.Min(x, y); i++)
```

Comment:
It is generally more efficient to go from the largest number to the smallest when searching for a divisor, because it reduces the number of iterations of the loop. For example, if you are searching for a divisor of a large number n, starting at the largest possible divisor n/2 and decreasing i by 1 each iteration will require significantly fewer iterations than starting at 2 and increasing i by 1 each iteration.